packages/next/src/storage.ts

```diff
@@ -0,0 +1,29 @@
+import { Storage, StorageKey } from '@logto/node';
+
+import { NextRequestWithIronSession } from './types';
+
+export default class NextStorage implements Storage {
+  constructor(private readonly request: NextRequestWithIronSession) {}
+
+  async setItem(key: StorageKey, value: string) {
+    this.request.session[key] = value;
+  }
+
+  getItem(key: StorageKey) {
+    const value = this.request.session[key];
+
+    if (typeof value === 'string') {
+      return value;
+    }
+
+    return null;
```

Comment:
will it be better to return `String(value)` except `value === undefined`? e.g.

```ts
if (value === undefined) {
   return;
}

return String(value);
```

Reply:
updated